cli/cmd/compliance_aws.go

```diff
@@ -243,6 +245,135 @@ To show recommendation details and affected resources for a recommendation id:
 			return nil
 		},
 	}
+
+	// complianceAwsDisableReportCmd represents the disable-report sub-command inside the aws command
+	// experimental feature
+	complianceAwsDisableReportCmd = &cobra.Command{
+		Use:     "disable-report <report_type>",
+		Aliases: []string{"disable"},
+		Hidden:  true,
```

Comment:
Even though these are all hidden commands, please add `Short` and `Long` descriptions. If you add them, they won't get documented, but they will be lost on what these commands do if someone finds them.